### PR TITLE
PiE On the Road page built, initial styling, forms coming soon

### PR DESCRIPTION
**Pull request by kev-chien**
Referencing this doc https://docs.google.com/document/d/1j2l-RHxzLg7wJ_b9xPTqXun5TNnxsrBcmD2npFrzH0w/edit

Should I move "Questions?" to the very bottom?

### CODE REVIEW

**Comment by scottnuma** [comment]
Moving questions to the very bottom sounds good - perhaps integrated with the FAQs

File: programs/fall-workshops.md [hunk 1]
```diff
@@ -1,50 +1,155 @@
...
+
+<!-- TODO: make visible later
+Each workshop has a total of 30 seats available. All students are welcome to attend any workshop, but we recommend the workshop at a high school closest to you. Students must register to attend. Should registration fill, we will place students on a waitlist in case a seat opens.
+-->
+
+PiE on the Road will be held at the following locations and times. Registration forms will be released shortly.
+
+Questions? Contact us at <mailto:education@pioneers.berkeley.edu>.
+
+<!--<h1 style="text-align: center">Events</h1>-->
```
Comment (scottnuma) [inline programs/fall-workshops.md]: Looks like these comments are no longer needed

File: programs/fall-workshops.md [hunk 2]
```diff
@@ -1,50 +1,155 @@
...
+</div>
+</div>
 
-Questions, comments, concerns? Email us at <mailto:education@pioneers.berkeley.edu>.
 
-<div class="text-center">
-    <a href="/programs/fall-workshops-registration.html" class="btn btn-primary">Register for Workshop</a>    
+<div class="pie-jumbotron">
+<div class="row">
+    <div class="col-xs-12"><h2>PiE on the Road @CCPA</h2></div>
```
Comment (scottnuma) [inline programs/fall-workshops.md]: I'm not sure how many people will recognize CCPA for a heading - can you expand out to Coliseum College Prep Academy?

Comment (kev-chien) [reply programs/fall-workshops.md]: addressed

**Comment by scottnuma** [comment]
Something minor, but I'd also like to keep the URL link the same - we've already shipped the link for publicity.

https://pioneers.berkeley.edu/programs/fall-workshops.html

**Comment by kev-chien** [comment]
@scottnuma The url is the same in the PR I believe?

**Comment by scottnuma** [comment]
Yup, as current is good, just want to make sure we're all aware

**Pull request merged**